resources/views/hardware/checkout.blade.php

```diff
@@ -61,6 +60,9 @@
                 @include ('partials.forms.edit.location-select', ['translated_name' => trans('general.location'), 'fieldname' => 'assigned_location', 'style' => 'display:none;', 'required'=>'true'])
 
 
+                @include ('partials.forms.edit.department-select', ['translated_name' => trans('general.department'), 'fieldname' => 'assigned_depaetment', 'style' => 'display:none;', 'required'=>'true'])
```

Comment:
You have a typo at `'fieldname' => 'assigned_depaetment'`. Should be `'fieldname' => 'assigned_department'`